_includes/picard-metric-definitions.html

```diff
@@ -57,6 +57,9 @@
 <li><a href="#TargetedPcrMetrics">TargetedPcrMetrics</a>: Metrics class for targeted pcr runs such as TSCA runs</li>
 </ol>
 </section>
+<section>
+<p><b>Please note: Although metrics are indicated as percents, they are actually fractions.</b></p>
```

Comment:
This should go at the top of the page; otherwise people might scroll down only until they find the tool they're interested in, click on it and miss the warning. It can even be on the same line as that first "Click on a metric to see a description of its fields." sentence.